metabase/metabase
# temporarily hiding group into other

Closes https://github.com/metabase/metabase/issues/50510

### Description

Temporarily hiding [group into other](https://www.notion.so/metabase/Allow-for-grouping-large-numbers-of-series-into-other-61660fc5206a48ed8ce12a19c2e6ef96) feature. Original feature PR: https://github.com/metabase/metabase/pull/48265
Instead of unmerging I disable the newly added settings: `isValid` is always false which makes them use the default values so even already existing values will be ignored.

### How to verify

- Create a bar chart with two dimensions and many series
- Ensure there is no "Group into Other" functionality

### Checklist

- [n/a] Tests have been added/updated to cover changes in this PR
